src/phoenix/core/model_schema_adapter.py

```diff
@@ -107,8 +109,19 @@ def create_model_from_datasets(*datasets: Optional[Dataset]) -> Model:
         prediction_score=_take_first_str(prediction_scores),
         actual_label=_take_first_str(actual_labels),
         actual_score=_take_first_str(actual_scores),
-        features=chain(features, translated_embeddings),
-        tags=tags,
+        features=chain(
+            *_guess_vectors_vs_scalars(
+                features,
+                *map(itemgetter(1), named_dataframes),
+            ),
+            translated_embeddings,
+        ),
```

Comment:
doesn't `chain` combine the lists? Don't we only want the `List[str]`?

Reply:
Oh hmm, maybe I didn't realize we ingest embeddings as features in this way.

Reply:
yea, features and embeddings are orthogonal properties